LautFM/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "author": {
+        "name": "MauriceX",
+        "id": "268768664411439115"
+    },
+    "service": "LautFM",
+    "description": {
+        "en": "Laut.FM is a Website to create your own Music station or listening to music.",
+        "de": "Laut.FM ist eine Webseite um dein eigenes Radio zu erstellen oder um Musik zu hören."
+
+    },
+    "url": [
+        "laut.fm"
+            
+    ],
+    "version": "1.0",
+    "logo": "https://i.imgur.com/rncpOUV.png",
+    "thumbnail": "https://i.imgur.com/FZh7OFc.png",
+    "color": "#373b4a",
+    "tags": [
+        "Music",
+		"Radio",
+		"Station"
+    ],
+    "iframe": false,
```

Comment:
iframe not required

Reply:
Iframe was removed and the blank space was removed too.